app/models/solr_document.rb

```diff
@@ -28,4 +26,12 @@ class SolrDocument
 
   # Do content negotiation for AF models.
   use_extension(Hydra::ContentNegotiation)
+
+  # overrides the MimeTypes included by CurationConcerns::SolrDocumentBehavior in order to add psd types.
+
+  module Hydra::Works::MimeTypes::ClassMethods
```

Comment:
Finding the right place to override something from the Hydra stack is definitely something I'm wrestling with right now in general. It's included in CC here, in its SolrDocumentBehavior: https://github.com/projecthydra/curation_concerns/blob/613651ef375316d779e4d5f40d608344472cdcb8/app/models/concerns/curation_concerns/solr_document_behavior.rb